runtime/themes/fleet_dark.toml

```
@@ -4,128 +4,161 @@
-"markup.list" = "pink" # .unnumbered / .numbered
-"markup.list.numbered" = "cyan"
-"markup.list.unnumbered" = "cyan"
+"markup.heading" = { fg = "Cyan", modifiers = ["bold"] } # .marker / .1 / .2 / .3 / .4 / .5 / .6
+"markup.list" = "Pink" # .unnumbered / .numbered
+"markup.list.numbered" = "Cyan"
+"markup.list.unnumbered" = "Cyan"
 # "markup.link" = "green"
-"markup.link.url" = { fg = "pink", modifiers = ['italic', 'underlined'] }
-"markup.link.text" = "cyan"
+"markup.link.url" = { fg = "Pink", modifiers = ['italic', 'underlined'] }
+"markup.link.text" = "Cyan"
 "markup.link.label" = "purple"
-"markup.quote" = "pink"
-"markup.raw" = "pink"
-"markup.raw.inline" = "cyan" # .completion / .hover
-"markup.raw.block" = "pink"
+"markup.quote" = "Pink"
+"markup.raw" = "Pink"
+"markup.raw.inline" = "Cyan" # .completion / .hover
+"markup.raw.block" = "#EB83E2"
 
-"diff.plus" = "diff_plus"
-"diff.minus" = "red_accent"
-"diff.delta" = "blue_accent"
+"diff.plus" = "Green 50"
+"diff.minus" = "Red 50"
+"diff.delta" = "Blue 80"
 
 # ui specific
-"ui.background" = { bg = "background" } # .separator
-"ui.cursor" = { bg = "dark_gray", modifiers = ["reversed"] } # .insert / .select / .match / .primary
-"ui.cursor.match" = { fg = "light", bg = "selection" } # .insert / .select / .match / .primary
-"ui.cursorline" = { bg = "darker" }
-"ui.linenr" = "dark_gray"
-"ui.linenr.selected" = { fg = "light", bg = "darker" }
-"ui.statusline" = { fg = "light", bg = "darker" } # .inactive / .normal / .insert / .select
-"ui.statusline.inactive" = { fg = "dark", bg = "darker" }
-"ui.statusline.normal" = { fg = "lightest", bg = "darker"}
-"ui.statusline.insert" = { fg = "lightest", bg = "blue_accent" }
-"ui.statusline.select" = { fg = "lightest", bg = "orange_accent" }
-"ui.popup" = { fg = "light", bg = "darkest" } # .info
-"ui.window" = { fg = "dark", bg = "darkest" }
-"ui.help" = { fg = "light", bg = "darkest" }
-"ui.text" = "light" # .focus / .info
-"ui.text.focus" = { fg = "lightest", bg = "focus" }
-"ui.virtual" = "dark" # .whitespace
-"ui.virtual.ruler" = { bg = "darker"}
-"ui.menu" = { fg = "light", bg = "darkest" } # .selected
-"ui.menu.selected" = { fg = "lightest", bg = "focus" } # .selected
-"ui.selection" = { bg = "darker" } # .primary
-"ui.selection.primary" = { bg = "selection" }
-"hint" = "blue"
-"info" = "yellow_accent"
-"warning" = "orange_accent"
-"error" = "red_error"
-"diagnostic" = { modifiers = [] }
-"diagnostic.hint" = { underline = { color = "light", style = "curl" } }
-"diagnostic.info" = { underline = { color = "blue_accent", style = "curl" } }
-"diagnostic.warning" = { underline = { color = "yellow_accent", style = "curl" } }
-"diagnostic.error" = { underline = { color = "red_error", style = "curl" } }
+"ui.background" = { bg = "Gray 10" } # .separator
+"ui.statusline" = { fg = "Gray 120", bg = "Gray 10" } # .inactive / .normal / .insert / .select
+# "ui.statusline.normal" = { fg = "lightest", bg = "darker"}
```

I commented out these since I didn't know what would be good colors.

---

```suggestion
"ui.statusline" = { fg = "Gray 120", bg = "Gray 20" }
"ui.statusline.insert" = { bg = "Blue 80" }
"ui.statusline.select" = { bg = "Yellow 40" }
```

I think it's important to separate `fleet` features from `helix` features and just make an educated guess at what people will prefer. Fleet relies heavily on blue, so I think it will be a safe and familiar color for insert mode. 

Yellow 40 or Green 40 might work too. The yellow is darker and warmer and reminds me of vim, but the green might be more in the style of fleet. What do you think?